napari/_qt/qt_resources/styles/00_base.qss

```diff
@@ -83,7 +83,10 @@ QPushButton:checked {
 }
 
 QPushButton:disabled {
-  background-color: {{ background }};
+  background-color: {{ opacity(foreground, 75) }};
+  border: 1px solid;
+  border-color: {{ foreground }};
+  color: {{ opacity(text, 90) }};
```

Comment:
```suggestion
  background-color: {{ opacity(foreground, 120) }};
  border: 1px solid;
  border-color: {{ foreground }};
  color: {{ opacity(text, 160) }};
```

Could you check these values, and how they look to you?

Reply:
Bah, I posted in the wrong spot.
For me on Dark Mode the disabled is less clearly disabled, if there is just one button that's disabled and no reference enabled button. So the user could possibly not realize it. So I prefer the original, but we can play with the values some more.